neptune/codelists_test.go

```diff
@@ -359,3 +359,80 @@ func TestGetCodes(t *testing.T) {
 		})
 	})
 }
+
+func TestGetCode(t *testing.T) {
+	Convey("Given a database that will return that the Code exists", t, func() {
+		poolMock := &internal.NeptunePoolMock{GetCountFunc: internal.ReturnOne}
+		db := mockDB(poolMock)
+		Convey("When GetCode() is called", func() {
+			unusedCodeList := "unused-code-list"
+			unusedEdition := "unused-edition"
+			unusedCode := "unused-code"
+			code, err := db.GetCode(context.Background(), unusedCodeList, unusedEdition, unusedCode)
+			Convey("Then no error should be returned", func() {
+				So(err, ShouldBeNil)
+			})
+			Convey("Then the driver GetCount function should be called once", func() {
+				calls := poolMock.GetCountCalls()
+				So(len(calls), ShouldEqual, 1)
```

Comment:
Minor comment but there is a `ShouldHaveLength` assertion in goconvey:
```
So(calls, ShouldHaveLength, 1)
```
No functional difference but just an FYI.

Reply:
thanks